src/Config.js

```diff
@@ -1,6 +1,12 @@
 export default class Config {
   static get platformUrl() {
-    return process.env.TRELLIS_BASE_URL || 'http://platform:8080'
+    // If configured in environment (hence, prod-like), use that value
```

Comment:
I'm still not  entirely convinced about the advantages having a class for `Config` is necessary for functional javascript

Reply:
@jermnelson I'd be happy to put in a follow-up PR that dumps the bespoke `Config` class in favor of the node `config` gem. It seems to me that our projects have already been moving in that direction. Is that cool by you?

Reply:
Does this suggest too that https://github.com/LD4P/sinopia_editor/issues/457 should be in scope? These are our last two codebases, AFAIK, that use a custom `Config` class.

Reply:
The bespoke `Config` class here is fine for now; it works and is tested. I don't think it is critical path but maybe better noted as a future fix if you want to make a ticket.

Reply:
@jermnelson OK! I'll write it up and tag it as "future"